internal/services/automation/automation_variable_string_resource_test.go

```diff
@@ -41,7 +41,7 @@ func TestAccAutomationVariableString_complete(t *testing.T) {
 				check.That(data.ResourceName).Key("description").HasValue("This variable is created by Terraform acceptance test."),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep("value"),
```

Comment:
since this is a totally different scenario (with different conditions) rather than adding this to the update/complete tests, we should add an `encrypted` test that verifies we can both create and update a variable where this is encrypted - can we fix this?

```suggestion
		data.ImportStep(),
```